lib/mayaUsd/resources/ae/usd-shared-components/src/python/usdSharedComponents/collection/expressionRulesMenu.py

```diff
@@ -9,15 +9,28 @@
 
 from pxr import Usd
 
+# TODO: support I8N
 EXPAND_PRIMS_MENU_OPTION = "Expand Prims"
 EXPAND_PRIMS_PROPERTIES_MENU_OPTION = "Expand Prims and Properties"
 EXPLICIT_ONLY_MENU_OPTION = "Explicit Only"
+kIncludeExcludeLabel = "Include/Exclude"
```

Comment:
minor: I know this is maya convention, but we probably would want to keep it capitalized for consistency in this case

Reply:
Ah, right I wrote these a few days ago, before the other review. We can change them afterward